### ISSUE TITLE
Etesync app shows wrong birthday date although it is correctly saved in the vcard

### ISSUE THREAD

**Issue by dcasado** [etesync/android#56]
As you can see it shows 6th of January instead of 7th of January.
![photo_2019-01-07_12-58-14](https://user-images.githubusercontent.com/10188511/50767119-ee4fdf00-127b-11e9-9703-7d3f2507852a.jpg)
![photo_2019-01-07_12-58-03](https://user-images.githubusercontent.com/10188511/50767125-f1e36600-127b-11e9-805b-ca27073d4010.jpg)

Device: Oneplus 6
Android Version: 9

**Comment by tasn** [comment]
Oops. :)

At least I assume EteSync works and it's just the in-app preview that's broken?

Will have to take a look into that.

**Comment by tasn** [comment]
Oops, didn't mean to close.

**Comment by dcasado** [comment]
Yes, synchronisation works well and in the rest of apps (Calendar and contacts) the date is correct.

**Comment by tasn** [comment]
OK, thanks for letting me know!

**Comment by tasn** [comment]
Fixed. Was an issue with timezone handling. Essentially, it would parse the time in your current timezone, but would format it in UTC. Dates without time have the time set to 00:00, and since when it's 00:00 on the 7th in Japan it's the 6th in the US, it was showing it wrong.

Thanks again for reporting.
